DirectX9/Direct3D9Functions.h

```diff
@@ -1,27 +1,27 @@
 
```

Comment:
As mentioned earlier, all changes to the DX9 project are rejected pending the merge of Dave's DX9 port. That said, the DX9 port does require some work to bring it more inline with the DX11 architecture, so some of these changes might well be wanted at a later date.